src/components/TodoContainer.js

```diff
@@ -1,77 +1,79 @@
-/* eslint-disable */
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
```

Comment:
Dear @devMahmoud disabling all of them is like disabling the whole file. Try to correct the ESLint errors in following projects